### DISCUSSION TITLE
Update to Manifest v3 for Chrome Web Store

### DISCUSSION THREAD

**Discussion by marshalljp** [gildas-lormeau/SingleFile#1352] [Q&A] [answered]
Google is eventually going to move completely away from allowing extensions that use manifest v2 in the web store. They already will not allow for new manifest v2 extensions from being put into the store. Is there any plans to migrate SingleFile to manifest v3?

**Comment by gildas-lormeau** [comment, marked as answer]
Yes, the extension is in fact ready to be migrated, see https://github.com/gildas-lormeau/SingleFile-MV3. 

Extension development with Manifest V3 is a real pain because of the bug mentioned on the project page. So I'm waiting until the last moment to migrate.

**Comment by marshalljp** [comment]
Oh thanks for the info! I really appreciate it and glad there is already a MV3 candidate ready.

**Reply by gildas-lormeau** [reply]
You're welcome!
